mcp/server/tools/src/services/SurveyService.ts

```
@@ -77,4 +85,81 @@
+            //   content: [
+            //     {
+            //       type: "text",
+            //       text: `question asked is: ${question}, matching user survey by ID's data is: ${JSON.stringify(survey, null, 2)}`
+            //     }
+            //   ]
+            // };
+        } catch (error: any) {
+            return {
+              content: [{ type: "text", text: `Error fetching your survey: ${error.message}` }]
+            };
+          }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)